ResearchUI/ResearchUI/iOS/Views/RSDStepNavigationView.swift

```diff
@@ -473,6 +461,40 @@ open class RSDTableStepHeaderView: RSDStepHeaderView {
         }
     }
     
+    /// Set up the vertical constraints for the views that should be stacked vertically, like the imageView, titleLabel, textLabel, and learnMoreButton. You will need to remove previous constraints and re-add the ones associated with your vertically stacked view.
+    /// @param currentLastView the lastView that was added to the navigation header, if no views are added in this function, this should be returned.
+    /// @return (firstView, lastView) firstView that had vertical constraints applied, nil if none were applied
+    ///                               lastView that had vertical constraints applied, nil if none were applied
+    open func updateVerticalConstraints(currentLastView: UIView?) -> (firstView: UIView?, lastView: UIView?) {
+        
+        var firstView: UIView?
+        var lastView: UIView?
```

Comment:
Should default to currentLastView:
`    var lastView: UIView? = currentLastView`